### PR TITLE
Big router refactor. No more route conflicts.

### PR DESCRIPTION
**Pull request by gabrielpoca**

### CODE REVIEW

**Comment by gabrielpoca** [comment]
What do you think @Jvxtv ? It's looks more complex, but I don't think it is.

File: collections/categories.coffee [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
     image: '/img/icon-add.png'
   },
   {
-    id: 'gas',
     value: 'gas'
     description: 'Gas',
     image: '/img/icon-add.png'
   },
   {
-    id: 'other',
```
Comment (joaojusto) [inline collections/categories.coffee:33]: Why the id removal?

Comment (gabrielpoca) [inline collections/categories.coffee:33]: What is it for? The value is also unique.

Comment (joaojusto) [inline collections/categories.coffee:33]: Ok .
I think it's ready to merge

**Pull request merged**